src/olympia/amo/mail.py

```diff
@@ -17,30 +17,39 @@ class DevEmailBackend(BaseEmailBackend):
     `settings.SEND_REAL_EMAIL` is disabled.
 
     BUT even if `settings.SEND_REAL_EMAIL` is disabled, if the targeted
-    email address is in the `settings.EMAIL_QA_ALLOW_LIST` list,
-    the email will be sent.
+    email address is in `dev_email` group users, the email will be sent.
     """
 
+    @staticmethod
+    def force_send_mail_group():
+        return 'Force Send Mail'
+
     def send_messages(self, messages):
         """Save a `FakeEmail` object viewable within the admin.
 
         If one of the target email addresses is in
-        `settings.EMAIL_QA_ALLOW_LIST`, it send a real email message.
+        `Group('force_send_mail_group')`, it send a real email message.
```

Comment:
```suggestion
        group named `Group.force_send_mail_group`, it send a real email message.
```

(Or something like that, to make it clearer - I'm not particularly happy with my suggestion either)

Reply:
Done.